<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigGuiClass</class>
 <widget class="QWidget" name="ConfigGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1367</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigGui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
}
QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QLabel
{
color: rgb(255, 255, 255);
font: 75 10pt &quot;黑体&quot;;
}

#back
{
border-radius:5px;
background-color: rgb(53, 53, 53);
}
#all
{
background-color: rgb(38, 38, 38);
}
#closeButton{
font: 150 14pt &quot;等线&quot;;
color: rgb(255, 255, 255);
background-color: rgb(53, 53, 53);
}
#title
{
color: rgb(0, 170, 255);
font: 75 12pt &quot;黑体&quot;;
}


#playslider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

#playslider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

#playslider::add-page:horizontal {
background: #fff;
border: 1px solid #077;
height: 10px;
border-radius: 4px;
}

#playslider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -5px;
margin-bottom: -5px;
border-radius: 4px;
}

#playslider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

#playslider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

#playslider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

#playslider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
  </property>
  <widget class="QWidget" name="back" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="add_config">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>新增配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>10</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="line" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1280</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(28, 28,28, 255), stop:1 rgba(68, 68, 68, 255));</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>    XMS微服务配置中心管理</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>671</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>微服务名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>端口</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>dir</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>20011</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>conf</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>20019</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="server_ip_edit">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="server_port_box">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;ADMUI3Lg&quot;;</string>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>20010</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>120</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>配置中心IP和端口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_button">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QListWidget" name="log_list_Widget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>381</width>
      <height>641</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>配置中心启动</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>连接成功</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="del_config">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_config">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>330</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑配置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="config_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigGuiClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1254</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>1366</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refresh_button</sender>
   <signal>clicked()</signal>
   <receiver>ConfigGuiClass</receiver>
   <slot>Refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1186</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>1366</x>
     <y>608</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_config</sender>
   <signal>clicked()</signal>
   <receiver>ConfigGuiClass</receiver>
   <slot>AddConf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1208</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>1320</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del_config</sender>
   <signal>clicked()</signal>
   <receiver>ConfigGuiClass</receiver>
   <slot>DelConf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1213</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>1433</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_config</sender>
   <signal>clicked()</signal>
   <receiver>ConfigGuiClass</receiver>
   <slot>EditConf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1179</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>1407</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>ConfigGuiClass</receiver>
   <slot>EditConf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>-82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddConf()</slot>
  <slot>Refresh()</slot>
  <slot>DelConf()</slot>
  <slot>EditConf()</slot>
 </slots>
</ui>
